common/web/utils/package.json

```diff
@@ -23,5 +23,9 @@
     "@keymanapp/keyman-version": "*",
     "@types/node": "^14.0.5",
     "typescript": "^4.5.4"
+  },
+  "type": "module",
+  "paths": {
+    "@keymanapp/keyman-version": "../../../node_modules/@keymanapp/keyman-version/build"
```

Comment:
Do we really need to point to the build folder? That's going to be squirmy for dev-time when it isn't built.

Reply:
... I don't even know what happened for that to be there.  Easy enough to revert, at least, and it doesn't seem to affect the build in any way.